packages/analytics-browser/src/index.ts

```diff
@@ -23,5 +23,6 @@ export const {
   track,
 } = client;
 export { runQueuedFunctions } from './utils/snippet-helper';
+export { GlobalScope } from './utils/global-scope';
```

Comment:
probably don't need this. i see that we'll move `globalScope` to `analytics-client-common` once we do the follow up task

Reply:
@kevinpagtakhan 
Initially I added it to `analytics-client-common` but I am not sure if it is required for `react-native` for example.
I moved it to `analytics-browser` because `marketing-analytics-browser` already uses code from `analytics-browser`